qcodes/utils/slack.py

```diff
@@ -0,0 +1,418 @@
+import os
+import tempfile
+from functools import partial
+from time import sleep
+import inspect
+from slacker import Slacker
+import threading
+import traceback
+
+from qcodes.plots.base import BasePlot
+from qcodes import config as qc_config
+from qcodes.instrument.parameter import _BaseParameter
+from qcodes import active_loop, active_data_set
+
+
+def convert_command(text):
```

Comment:
@nulinspiratie shut, never mind I totally misread the code.  Just catch the right Value error and then  I can merge!